packages/terra-heading/src/Heading.scss

```diff
@@ -0,0 +1,44 @@
+.terra-Heading {
+  font-weight: 700;
```

Comment:
For the base heading, I think it's better to just use resource styles, and then create classes for any styles we opt into, like we are doing for the italic class. Then in your component, I would just make the boldness prop defaulted to bold if you need